### ISSUE TITLE
Breadcrumbs in Backend

### ISSUE THREAD

**Issue by V15** [contao/contao#6027]
### Description

Sowohl in der **Seitenstruktur**, als auch in den **Artikeln** und **Inhaltselementen** wird oben auf der Seite keine eindeutige Information angezeigt, wo man sich genau im Seitenbaum befindet. Dadurch ist es schwierig zu erkennen wo eine Seite, ein Artikel oder ein Inhaltselement bearbeitet wird.

Wenn man die Ansicht über Knotenpunkte filtert existiert eine nachvollziehbare Breadcrumb.

**Beispiel: Knotenpunkt-Filterung bei Seiten**

![seiten-01](https://user-images.githubusercontent.com/4067828/236669202-71b9d5cc-30f8-428b-9c03-8983763fc67e.png)

**Beispiel: Bearbeitung bei Seiten (Auswahl der Seite durch Stiftsymbol)**

![seiten-02](https://user-images.githubusercontent.com/4067828/236669205-66d5f37e-1e80-4516-9d7e-107451b8f795.png)

Oberhalb wird nur „Seiten > Seite ID 51“ angezeigt.
Keine Informationen über Parent-Elemente und keine Möglichkeit ggf. zu navigieren.

**Wunsch-Darstellung**:  Seiten › Seiten-Startpunkt [Name / Sprache] › Seiten-Name › Unterseite

---

Das gleiche gilt für die Artikel.

**Beispiel: Filterung bei Artikel**

![artikel-01](https://user-images.githubusercontent.com/4067828/236669188-e277c952-1812-488b-ad63-0b08fe2f4b5c.png)

**Beispiel: Bearbeitung bei Artikel**

![artikel-02](https://user-images.githubusercontent.com/4067828/236669227-29a04349-5363-46f2-b774-9daf05522bf5.png)

Oberhalb wird nur „Artikel > Artikel ID 66,“ angezeigt.
Keine Informationen über Parent-Elemente und keine Möglichkeit ggf. zu navigieren.

**Wunsch-Darstellung**:  Artikel › Seiten-Startpunkt [Name / Sprache] › Seiten-Name › Unterseite › Artikel-Name

---

Das gleiche gilt für die Inhaltselemente.

**Beispiel: Ansicht der Inhaltselemente**

![inhalt-01](https://user-images.githubusercontent.com/4067828/236669239-b97dfed9-8cf1-4d35-b5a9-165252d67165.png)

Oberhalb wird nur „Artikel > Home > Inhaltselement“ angezeigt.
Keine Informationen über Parent-Elemente und keine Möglichkeit ggf. zu navigieren.

**Wunsch-Darstellung**:  Artikel › Seiten-Startpunkt [Name / Sprache] › Seiten-Name › Unterseite › Artikel-Name › Inhaltselement-Name

Ein gutes Beispiel sind die Nachrichten, dort ist es genauso umgesetzt.

**Beispiel: Ansicht der Inhaltselemente einer Nachricht:**

![news-01](https://user-images.githubusercontent.com/4067828/236669254-0ba5d6cc-826e-4e1d-b9dc-6e0df9baa2a9.png)

**Beispiel: Ansicht eines Inhaltselements einer Nachricht:**

![news-02](https://user-images.githubusercontent.com/4067828/236669264-52e081f6-cdf5-4fb2-8337-d4c0c9fa22ee.png)

**Wunsch**: Bei MouseOver die einzelnen Knotenpunkte ausgeschrieben anzeigen (Tooltip).

Zusätzlich wäre es auch eine Option, z.B. über ein „3-Punkte-Symbol“ o.ä. den Pfad aufzuklappen und mehrzeilig oder untereinander als Hierarchie darzustellen.

(As discussed on Contao Camp 2023 / https://github.com/contao-community-alliance/contao-camps/blob/main/2023/camp-2023.md#samstag-06052023)

**Comment by zoglo** [comment]
@ausi is this easy to implement right now 🤔 ?

**Comment by ausi** [comment]
Yes, but it needs to be more compact because otherwise the breadcrumbs would get too long. But we could implement some kind of tooltip when you hover/click on the article or page name.

**Comment by zoglo** [comment]
See #9147
